source/common/http/http_server_properties_cache_impl.h

```diff
@@ -23,9 +23,9 @@ namespace Http {
 // A cache of HTTP server properties.
 // This caches
 //   - alternate protocol entries as documented here: source/docs/http3_upstream.md
```

Comment:
nit: "alternate" => "Alternate" (and "." at the end) to be a complete sentence like the others in this list.